libraries/importers/pom.xml

```diff
@@ -25,7 +25,6 @@
 
 	<properties>
 		<artifact.deploy.dir>${liferay.deploy.dir}</artifact.deploy.dir>
-		<maven.test.skip>true</maven.test.skip>
```

Comment:
@smrutis1 revert this change

Reply:
`importers` module test case is fixed [ref](https://github.com/eclipse-sw360/sw360/actions/runs/9873727882/job/27270450726?pr=2501#step:12:19940). Hence its removed. Let me know.

Reply:
Yep, but the PR still not passing, should we merge it anyway ?